packages/blade/src/components/Table/Table.web.tsx

```diff
@@ -504,7 +537,7 @@ const _Table = <Item,>({
           )}
           {toolbar}
           <StyledReactTable
-            role="table"
+            role={isVirtualized ? 'grid' : 'table'}
```

Comment:
Why are we changing the accessibility role to grid when it's virtualized? A table should be table only regardless of virtualized or not?